setup/helpers/sandbox.js

```diff
@@ -1,17 +1,22 @@
+"use strict";
+
 const path = require("path");
+const CONFIG = require("./config");
 const { displayProjects } = require("./projects");
 
-const ENCODED_DIR = remote.app.getAppPath().replace(/ /g, "%20");
+const ENCODED_DIR = CONFIG.REMOTE_APP.replace(/ /g, "%20");
 
 function displaySandbox() {
-    let sandboxes = [];
+    const sandboxes = [];
 
     sandboxes[0] = {
```

Comment:
@Pexeus is there any reason that you used array here? and passing an object into it? can the sandboxes array take more than 1 object? I was confused about that one

Reply:
Its because im not sure if I maybe wanna use the sandbox feature for other stuff in the future. maybe adding some demo projects or stuff like that. But at the time, there is no reason for it to be an array.

Something ive seen in your code: relative file paths like `./data/whatever` are broken as soon as the program gets packaged. I solved that by using `remote.app.getAppPath() + "file/path"`

But again, thank you so much for the work man!

Reply:
oh i see, i am going to create new pull request and update those path to use `remote.app.getAppPath()`, i’ve seen you were using those paths so I’ve just made config file for those relative paths